disputer/index.js

```diff
@@ -81,11 +81,13 @@ async function run(logger, address, pollingDelay, priceFeedConfig, disputerConfi
 
     // The EMP requires approval to transfer the disputer's collateral tokens in order to dispute a liquidation.
     // We'll set this once to the max value and top up whenever the bot's allowance drops below MAX_INT / 2.
+    await gasEstimator.update();
```

Comment:
The main drawback of this PR is that the GasEstimator is now getting updated one extra time, likely a redundant update between the allowance getting increased and the bot starting.

Reply:
This shouldn't be a big deal since we throttle the GasEstimator updates to once every 60 seconds by default, which means that if the approval transactions aren't needed, we most likely won't update again during the first iteration.

Reply:
Great point!